litestar/serialization.py

```diff
@@ -35,6 +35,7 @@
     from litestar.types import TypeEncodersMap
 
 __all__ = (
+    "DEFAULT_TYPE_ENCODERS",
```

Comment:
```suggestion
```

I don't think we actually want to export those. They're undocumented on purpose.